src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx

```diff
@@ -51,4 +54,19 @@ describe('facilities <FacilityPatientSatisfactionScoresWidget>', () => {
 
     tree.unmount();
   });
+
+  it('should NOT render facility patient satisfaction scores section when 0 values', () => {
+    const tree = shallow(
+      <FacilityPatientSatisfactionScoresWidget
+        loading={false}
+        facility={mockFeedbackZeroValues.data[0]}
+      />,
+    );
+    expect(tree).to.be.empty;
+    expect(tree.find('LoadingIndicator').exists()).to.be.false;
```

Comment:
Can we use `not.to.exist` here instead?

Reply:
So I changed `expect(tree.find('FacilityPatientSatisfactionScoresWidget').exists()).to.be` to `expect(tree.find('FacilityPatientSatisfactionScoresWidget')).not.to.exist;` But it doesn't pass

Reply:
``` 1) facilities <FacilityPatientSatisfactionScoresWidget>
       should NOT render facility patient satisfaction scores section when 0 values:
     AssertionError: expected {} to not exist
      at Context.<anonymous> (src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx:67:5)
      at Test.Runnable.run (node_modules/mocha-snapshots/src/index.js:19:22)
      at processImmediate (internal/timers.js:461:21)
```

Reply:
OK, np, thanks. We can leave as is.